apps/dot-voting/contracts/DotVoting.sol

```diff
@@ -281,7 +288,7 @@ contract DotVoting is ADynamicForwarder, AragonApp {
     * @return True if the vote is elligible for execution.
     */
     function canExecute(uint256 _voteId) public view isInitialized returns (bool) {
-        require(_voteId < voteLength, "Vote ID outside of current vote range");
+        require(_voteId < voteLength, VOTE_LENGTH_ERROR); // solium-disable-line error-reason
```

Comment:
I don't think we need the `solium-disable-line` comment here

Reply:
Correct